src/graphql/resolvers.ts

```diff
@@ -59,6 +60,54 @@ export const resolvers = {
         throw new CustomError('Could not create user: ' + error.message, 500)
       }
     },
+
+    login: async (_, args) => {
+      const loginInput = args.data
+      let searchedUser: User
+
+      try {
+        searchedUser = await prisma.user.findUnique({
+          where: {
+            email: loginInput.email,
+          },
+        })
+      } catch (error) {
+        throw new CustomError(
+          'Could not find user: ' + error.message,
+          error.extensions.code
+        )
+      }
+
+      const passwordMatches: boolean = await bcrypt.compare(
+        loginInput.password,
+        searchedUser.password
+      )
+
+      if (!passwordMatches) {
+        throw new CustomError('Password does not match', 401)
+      } else {
+        try {
+          const userInfo = {
+            id: searchedUser.id,
+            name: searchedUser.name,
+            email: searchedUser.email,
+            birthDate: searchedUser.birthDate,
+          }
+          const token = ''
+          const loginResponse = {
+            user: userInfo,
+            token: token,
```

Comment:
```suggestion
            token,
```
quando a chave tem o mesmo nome da variável pode deixar só a chave